tools/clang/unittests/HLSLTestLib/FileCheckerTest.cpp

```diff
@@ -139,10 +140,25 @@ FileRunCommandResult FileRunCommandPart::RunFileChecker(const FileRunCommandResu
   auto args = strtok(Arguments);
   for (const std::string& arg : args) {
     if (arg == "%s") hasInputFilename = true;
-    else if (arg == "-input=stderr") {
-      t.InputForStdin = Prior->StdErr;
-      t.AllowEmptyInput = true;
-    } else if (strstartswith(arg, checkPrefixStr))
+    else if (arg == "-input=stderr") t.InputForStdin = Prior ? Prior->StdErr : "";
```

Comment:
This lost the assignment of `t.AllowEmptyInput = true`, which is necessary for certain tests in certain environments